sdk/core/Azure.Core/CHANGELOG.md

```diff
@@ -1,14 +1,6 @@
 # Release History
 
-## 1.34.0-beta.1 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+## 1.34.0 (2023-07-11)
```

Comment:
If we don't have specific items to call out, can we add a generic line item for something like "Minor bug fixes and enhancements"?